scripts/generate-clients.mjs

```diff
@@ -114,5 +114,6 @@ kinobi.accept(
   renderRustVisitor(path.join(rustClient, "src", "generated"), {
     formatCode: true,
     crateFolder: rustClient,
+    toolchain: "+nightly-2023-10-05",
```

Comment:
I think we're starting to diverge here.

https://github.com/paladin-bladesmith/rewards-program/blob/8934926b08280fd6f1abb1b8f17dba761f573f54/scripts/generate-clients.mjs#L36

Reply:
You way is definitely better, I forgot to use the helper here.